### ISSUE TITLE
Skethcup plugin error at startup: path name contains null byte

### ISSUE THREAD

**Issue by macumber** [openstudiocoalition/openstudio-sketchup-plugin#73]
See https://github.com/NREL/OpenStudio/issues/4322

**Comment by macumber** [comment]
Did your other user name have any non ascii characters like ñ in it?  That might be the issue

You can try deleting all the OpenStudio related preferences from:

`C:\Users\<your username>\AppData\Local\SketchUp\SketchUp 2021\SketchUp\PrivatePreferences.json`

**Comment by macumber** [comment]
Did your other user name have any non ascii characters like ñ in it?  That might be the issue

You can try deleting all the OpenStudio related preferences from:

`C:\Users\<your username>\AppData\Local\SketchUp\SketchUp 2021\SketchUp\PrivatePreferences.json`

**Comment by gpmendoza** [comment]
> 
> 
> Did your other user name have any non ascii characters like ñ in it? That might be the issue
> 
> You can try deleting all the OpenStudio related preferences from:
> 
> `C:\Users\<your username>\AppData\Local\SketchUp\SketchUp 2021\SketchUp\PrivatePreferences.json`

YES!!!!! that solved it!!!! erasing the OpenStudio preferences file! I just tried and it´s working again, thank you so much!
